geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -2164,4 +2164,16 @@
    * <p>
    */
   String THREAD_MONITOR_TIME_LIMIT = "thread-monitor-time-limit-ms";
+  /**
+   * The static String definition of the <i>"check-registered-keys-interval-ms"</i> property <a
+   * name="check-registered-keys-interval-ms"/a>
+   * <p>
+   * <U>Description</U>: Defines the time interval (in milliseconds) with which checkRegisteredKeys
+   * function
+   * can be called.
```

Comment:
Minor nitpick, shouldn't this be on the line above?  There seems to be an unnecessary line break.